src/sdk/models/entityTypes.ts

```diff
@@ -7,10 +7,22 @@ export interface EntityType extends IEntityIdentifier<string>, IAuditedEntity {
 	kind?: 'pattern' | 'block_list' | 'transform' | 'classifier' | string;
 	category?: Category;
 	threshold?: number;
+	keywords?: EntityTypeDictionary;
```

Comment:
Same question as https://github.com/portalarchitects/skysync-cli/pull/181 Are we not going to create multiple EntityType classes like on back end? `keywords` property does not apply to `TransformEntityType`

cc: @bentatum, @gotvino, @smbecker